### PR TITLE
[UI] Fixed registry not sorting models alphabetically

### LINKED ISSUE #9401: [UI] Registery: not sorting models alphabetically
The Registry isn’t sorting models alphabetically, it seems.

### PR DESCRIPTION
**Pull request by Shiva953**
**Notes for Reviewers**

This PR fixes #9401 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by saurabh100ni** [comment]
Thanks for raising the PR @Shiva953,
Please add this in the [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit) 
as you item to discuss, if you would for the meshery development meet on 8:30pm IST and 9:00 am CT tomorrow.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
@ptdatta moving head w/o your review.

**Comment by leecalcote** [comment]
Thanks, @Shiva953 👏

**Comment by ptdatta** [comment]
@Shiva953 @leecalcote  most of the things are right just a couple of bugs which I fixed in [PR](https://github.com/meshery/meshery/pull/9420)

File: ui/components/MeshModelRegistry/MeshModelComponent.js [hunk 1]
```diff
@@ -104,10 +104,10 @@ const MeshModelComponent = ({
         model.components = components;
         model.relationships = relationships;
       });
-
       await Promise.all(componentPromises);
```
Comment (leecalcote) [inline ui/components/MeshModelRegistry/MeshModelComponent.js:107]: @aabidsofi19 Promise == bad, yes? // @ptdatta

Comment (Yashsharma1911) [reply ui/components/MeshModelRegistry/MeshModelComponent.js:107]: RTK can be a better option

**Pull request merged**